src/main/java/wolox/training/dto/BookDTO.java

```diff
@@ -0,0 +1,16 @@
+package wolox.training.dto;
+
+import java.io.Serializable;
+import lombok.Data;
+
+@Data
+public class BookDTO implements Serializable {
```

Comment:
Because I took this [example](https://www.oscarblancarteblog.com/2018/11/30/data-transfer-object-dto-patron-diseno/), that use this interface... But is not necessary, I can remove it